odoo_module_migrate/migration_scripts/migrate_120_130.py

```diff
@@ -23,6 +23,14 @@
     },
 }
 
+_TEXT_WARNINGS = {
+    ".py": {
+        r"\.env\.user\.company_id":
+        "[13] If the intended use is to get 'current active company'"
+        ", you need to change '.env.user.company_id' to '.env.company'",
+    },
+}
+
```

Comment:
Apart from that, LGTM

Reply:
Thanks for the suggestion :+1: I am trying to implement it. It seems to be working for the text warnings but not for the replacements. How can I check that it is picking up the yaml files?

Also, an issue that might be related: The last times I ran the utility it is not bumping the version in the manifest. Maybe it is not picking up that script as well? I have the version from `master`.

Reply:
loading yaml files happes here

https://github.com/OCA/odoo-module-migrator/blob/f0a3e30a10daac84b5fa4786afce02ce2e9c2813/odoo_module_migrate/base_migration_script.py#L55-L64

Replacements are tested by this rule, so it should work

https://github.com/OCA/odoo-module-migrator/blob/f0a3e30a10daac84b5fa4786afce02ce2e9c2813/odoo_module_migrate/migration_scripts/text_replaces/migrate_120_130/decorators.yaml#L2
https://github.com/OCA/odoo-module-migrator/blob/f0a3e30a10daac84b5fa4786afce02ce2e9c2813/tests/data_template/module_080/models/sale_order.py#L13-L16

https://github.com/OCA/odoo-module-migrator/blob/f0a3e30a10daac84b5fa4786afce02ce2e9c2813/tests/data_result/module_080_130/models/sale_order.py#L11-L14


The same about verion. That code was moved to new folders and seems working

https://github.com/OCA/odoo-module-migrator/blob/master/odoo_module_migrate/migration_scripts/python_scripts/migrate_allways/bump_version.py

Reply:
@yelizariev I made those changes and added another test.

Turns out the error I was getting was from the way the yaml files are loaded. If 2 files define different rules for the same file type on the same version, one would override the other. https://github.com/OCA/odoo-module-migrator/pull/53/commits/691e0a2b39ca0073b6c1a14bf8c154648576949b would allow to keep that option of splitting into several files.

The other issue I was getting of `bump_version` not being executed happens when I install from git (with `pipx install git+https://github.com/OCA/odoo-module-migrator.git`) and I can see that the yaml files are not being installed locally:
```
jota@jlap:~
➤ pipx install git+https://github.com/OCA/odoo-module-migrator.git@master --force
Installing to existing directory '/home/jota/.local/pipx/venvs/odoo-module-migrator'
  installed package odoo-module-migrator 0.3.2, Python 3.9.1
  These apps are now globally available
    - odoo-module-migrate
done! ✨ 🌟 ✨
jota@jlap:~
➤ ll .local/pipx/venvs/odoo-module-migrator/lib64/python3.9/site-packages/odoo_module_migrate/migration_scripts/
total 52
drwxr-xr-x. 3 jota jota 4096 fev 10 15:05 ./
drwxr-xr-x. 4 jota jota 4096 fev 10 15:05 ../
-rw-r--r--. 1 jota jota    0 fev 10 15:05 __init__.py
-rw-r--r--. 1 jota jota 2795 fev 10 15:05 migrate_080_090.py
-rw-r--r--. 1 jota jota  667 fev 10 15:05 migrate_080_allways.py
-rw-r--r--. 1 jota jota 1625 fev 10 15:05 migrate_090_100.py
-rw-r--r--. 1 jota jota 3335 fev 10 15:05 migrate_100_110.py
-rw-r--r--. 1 jota jota  416 fev 10 15:05 migrate_100_allways.py
-rw-r--r--. 1 jota jota 1886 fev 10 15:05 migrate_110_120.py
-rw-r--r--. 1 jota jota 2640 fev 10 15:05 migrate_120_130.py
-rw-r--r--. 1 jota jota 5010 fev 10 15:05 migrate_130_140.py
-rw-r--r--. 1 jota jota 1240 fev 10 15:05 migrate_allways.py
drwxr-xr-x. 2 jota jota 4096 fev 10 15:05 __pycache__/
```
Any idea of what could be happening?

Reply:
:thinking: 
maybe add [include_package_data](https://setuptools.readthedocs.io/en/latest/userguide/datafiles.html) to setup.py ?

Apparently, I didn't face this problem, because I use ``pip install -e ...``
```
       -e,--editable <path/url>
              Install a project in editable mode (i.e.  setuptools "develop mode") from a local project path or a VCS url.
 ```

Reply:
Yes, that fixed the issue!

With `pipx`, to install in editable mode it has to be from a local path, but now it works as expected. It might be broken on a lot of installations, though... The problem is just when installing from git, though, as that refactor is not included in the latest release...

Reply:
Don't forget to push updates for setup.py file :wink: